src/main.js

```diff
@@ -103,11 +113,17 @@ routeConfig(app, [
   new WebhookController(settings.webhookRepository),
 ]);
 
-app.use((request, response, next) => response.sendStatus(404));
```

Comment:
Yeah, I'm thinking that we move all the routing functions to another file. That will call `routeConfig`